pkg/oc/cli/admin/groups/sync/prune.go

```diff
@@ -215,8 +213,8 @@ func (o *PruneOptions) GetBlacklist() []string {
 	return o.Blacklist
 }
 
-func (o *PruneOptions) GetClient() userv1client.GroupInterface {
-	return o.GroupInterface
+func (o *PruneOptions) GetClient() userv1typedclient.GroupInterface {
```

Comment:
I would rather remove this method altogether, and just call o.GroupClient.Groups() at point of use

Reply:
It's more complicated than I want to and I don't want to deal with these complexities.